neo/Network/P2P/Payloads/WitnessScope.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Neo.IO;
+using Neo.IO.Json;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class WitnessScope : ISerializable, ICloneable<WitnessScope>
```

Comment:
Can this be combined with `CosignerUsage`?

Reply:
With `witnessScope` is easier for send this as a parameter in Sign methods

Reply:
Why do we need to send it as a parameter in `Sign()` method?

Reply:
Right, the logic was changed...

Reply:
Done!